README.md

```diff
@@ -58,12 +58,40 @@ If you also want to run the MySQL tests, ensure that MySQL is installed,
 and then type the following:
 
     mysql -uroot
-    create database hibernaterx;
+    create database hibernate-rx;
     create user "hibernate-rx" identified by 'hibernate-rx';
-    grant all on hibernaterx.* to "hibernate-rx";
+    grant all on hibernate-rx.* to "hibernate-rx";
 
 Finally, run `./gradlew test` from the `hibernate-rx` directory.
 
+### Podman
+
+Alternatively, you can start Postgres and MySQL using podman:
+
+* Postgres
+
+```
+  sudo podman run --ulimit memlock=-1:-1 -it --rm=true --memory-swappiness=0 --name HibernateTestingPGSQL -e POSTGRES_USER=hibernate-rx -e POSTGRES_PASSWORD=hibernate-rx -e POSTGRES_DB=hibernate-rx -p 5432:5432 postgres
+```
+
+* PSQL (Postgres client)
+
+```
+  sudo podman exec -it HibernateTestingPGSQL psql  -U hibernate-rx -W -d hibernate-rx
+```
+
+* MySQL
+
+```
+  sudo podman run --rm -it --name HibernateTestingMariaDB -e MYSQL_ROOT_PASSWORD=hibernate-rx -e MYSQL_DATABASE=hibernate-rx -e MYSQL_USER=hibernate-rx -e MYSQL_PASSWORD=hibernate-rx -p 3306:3306 mysql
+```
+
+* MySQL Client
+
+```
+  sudo podman exec -it HibernateTestingMariaDB mysql -U hibernate-rx -p
+```
+
```

Comment:
@DavideD Please spare us the use of `podman`. I tried that way and it didn't work at all. It's *way* easier to install Postgres or MySQL using a package manager like `brew` or whatever.

Reply:
Could we keep a note for `podman` users? It's quite handy for those on Linux.

+1 for it to not be the only way